python/dgl/utils/data.py

```diff
@@ -42,13 +45,20 @@ def scipy2tensor(spmat, idtype):
 
     Returns
     -------
-    (Tensor, Tensor)
-        Edge tensors.
+    (str, tuple[Tensor])
```

Comment:
Not necessarily.  It can also be a normal tuple.  `SparseAdjTuple` is a `namedtuple` which also has the same form (and can also be accessed with things like `data[0]`).